scripts/update-product-tags/update-product-tags.py

```diff
@@ -0,0 +1,156 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import requests
+import argparse
+import os
+import csv
+import random
+import uuid
+import requests
+from dotenv import load_dotenv
+load_dotenv()
+
+load_dotenv()  # take environment variables from .env.
```

Comment:
double load_dotenv()?
```suggestion

load_dotenv()  # take environment variables from .env.
```

Reply:
i really want to load the env